src/boot/lib/ast.ml

```diff
@@ -232,7 +232,9 @@ let rec map_tm f = function
   | TmMatch(fi,t1,p,t2,t3) ->
     f (TmMatch(fi,map_tm f t1,p,map_tm f t2,map_tm f t3))
   | TmUse(fi,l,t1) -> f (TmUse(fi,l,map_tm f t1))
-  | TmUtest(fi,t1,t2,tnext) -> f (TmUtest(fi,map_tm f t1,map_tm f t2,map_tm f tnext))
+  | TmUtest(fi,t1,t2,tusing,tnext) ->
+    let tusing_mapped = Option.map (fun t -> map_tm f t) tusing in
```

Comment:
I suspect `(fun t -> map_tm f t)` can be written as `(map_tm f)`

Reply:
Haha good point.